### DISCUSSION TITLE
Trust?

### DISCUSSION THREAD

**Discussion by Ebot2** [bridge-core/bridge.#461] [Q&A] [answered]
Attempting to run bridge_2.7.43_x64_en-US(https://bridge-core.app/guide/download/) flags windows defender as a virus. How can i trust this program, is there proof there isn't a trojan? I am not just stating this on a whim of luck iv seen this is many other reddits and even one in this GitHub page.

**Comment by outercloudstudio** [comment, marked as answer]
If you would like to compile the program from source, I'm happy to walk you through the process. The source code is all freely available for you to browse. We're not sure why the program had started triggering anti viruses. Many people use bridge, you can join our discord and ask around. Other than that I don't think there is much else I can do to reassure you.

**Reply by Ebot2** [reply]
I will try asking around as you said. thanks for the help
